index.js

```diff
@@ -164,6 +164,15 @@ exports.kinesisHandler = (records, opts = {}, context, callback) => {
       }
 
       if (error.name === 'HoldRequestConsumerError') {
+        if (error.errorType === 'empty-filtered-records') {
+          logger.warning(
```

Comment:
ahh yes, I just felt that I was using notice everywhere for basic logging on the logical flow. I find this situation to be unique because it is not an error that the filtered records returned an empty array. However, I do want to log that the execution of the promise chain WILL stop due to 0 records. As per the definitions, notice seems to fit the criteria. Will change thanks Kevin!